### DISCUSSION TITLE
EP never combine removes text from Icons

### DISCUSSION THREAD

**Discussion by PX03AFK** [valinet/ExplorerPatcher#1941] [Q&A] [answered]
I am very grateful for EP but I do have a small problem.  I use MS Outlook and if I run with icons combined with taskbar full Outlook presents individual icons for each email that I open.  If I then tell EP to always combine the icons lose their text.  Is this inevitable or should the text still be there?

**Comment by tallman69** [comment, marked as answer]
That behavior is by design.  As long as I can remember, in Windows always combining icons in the taskbar also removed the labels

**Comment by Poopooracoocoo** [comment]
7+ Taskbar Tweaker might help

**Reply by PX03AFK** [reply]
Thank you.

**Reply by PX03AFK** [reply]
I notice that 7+ Taskbar Tweaker says it doesn't work with W11 but I can't see if it will work with EP.

**Reply by tallman69** [reply]
If you have EP set to revert the taskbar back to the Windows 10 style, 7+Taskbar Tweaker may work.  See this blog:  https://ramensoftware.com/7-taskbar-tweaker-on-windows-11-with-windows-10s-taskbar.  

I have not personally tried 7+Taskbar Tweaker because I am a little Leary of using too many apps that alter the Windows 11 UI.  I currently use ExplorerPatcher and Open-Shell.

**Reply by PX03AFK** [reply]
Having asked the question I found several references online that suggest that 7+ Taskbar Tweaker will work with EP, I think because EP effectively runs the W10 taskbar.
